scripts/tasks/root-redirect-page.js

```diff
@@ -0,0 +1,13 @@
+module.exports = function (gulp, plugins, consts) {
+  return function () {
+    var fs = require('fs');
+
+    var outputPath = plugins.path.join(consts.DIST, 'index.html');
```

Comment:
Why basics? This is a root redirect file that will redirect from / to /version/docs . IMO it should be called index.html